src/dlgprefcontrols.cpp

```diff
@@ -276,9 +276,43 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
             ++j;
         }
     }
+	
+	ComboBoxTabletSkinconf->clear();
```

Comment:
Can you please verify that the code style follows http://www.mixxx.org/wiki/doku.php/coding_guidelines
eg: 
* no tabs
* opening brace at line end